### PR TITLE
Fixup license and copyright details

### PR DESCRIPTION
**Pull request by jasnell**
/cc @dankohn @aspiringarc ... please take a look, will land the PR once signed off

### CODE REVIEW

**Comment by dankohn** [comment]
Per https://www.linuxfoundation.org/blog/2020/01/copyright-notices-in-open-source-software-projects/ please remove the year, at least from the latter line:

```
Copyright 2020 HSEIreland
Copyright The COVID Green Contributors
```

File: README.md [hunk 1]
```diff
@@ -430,4 +430,18 @@ In order to upload/download diagnosis keys for exposure notifications, an applic
 
 ## License
 
-Licensed under [Apache v2.0](LICENSE)
+Copyright (c) 2020 Health Service Executive (HSE)
+Copyright (c) 2020 The COVID Green Contributors
```
Comment (jasnell) [inline README.md]:
```suggestion
Copyright (c) The COVID Green Contributors
```

**Comment by jasnell** [comment]
Done!

**Comment by dankohn** [comment]
Thanks. Could you please just give me another day to have our attorney review?

**Comment by jasnell** [comment]
No problem :-)

**Review by dankohn — APPROVED** [review]

**Pull request merged**